env/swagger.template

```diff
@@ -0,0 +1 @@
+SHOW_SWAGGER
```

Comment:
i don't think we should have a separate env file for swagger. Swagger is a part of the django, so it can be stored in backend.env